src/go/rpk/pkg/cli/acl/common.go

```diff
@@ -87,6 +89,12 @@ func unptr(str *string) string {
 	return *str
 }
 
+func PrefixRole(roles []string) {
+	for i, u := range roles {
+		roles[i] = rolePrefix + u
```

Comment:
Yeah, I waffled on this a bit. Since it's undocumented by design, it feels sort of arbitrary, but I think I'll just add the check. No reason to leave a footgun there.